src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1052 @@ metadata_pkg.(
     is_wrapper = false
 )
 
+"""
+$(MMI.doc_header(LinearRegressor))
+
+`LinearRegressor` assumes the target is a continuous variable
+whose conditional distribution is normal with constant variance, and whose
+expected value is a linear combination of the features. Linear coefficients
+are calculated using least squares.
+Options exist to specify a bias term.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Continuous`; check the scitype with `scitype(y)`
+
```

Comment:
We're missing that line that tells user how to train the machine: https://github.com/JuliaAI/MLJDecisionTreeInterface.jl/blob/d78f219775054d7d1bad318e9e8f0b0845f1f2ef/src/MLJDecisionTreeInterface.jl#L358